src/lib/types/experimental.ts

```diff
@@ -222,6 +223,13 @@ const flags: IFlags = {
                 '',
         },
     },
+    createdByUserIdDataMigration: {
+        name: 'createdByUserIdDataMigration',
```

Comment:
I think this is not needed
```suggestion
```

Reply:
Needed to change the whole thing to
`createdByUserIdDataMigration: parseEnvVarBoolean(
     process.env.CREATED_BY_USERID_DATA_MIGRATION,
     false,
),
`